fix: BROS-4: Sentry adds Baggage header to presigned urls and it breaks CORS for storages. Fix Sentry.ts

makseq opened the pull request:
# Problem
Sentry adds Baggage header to presigned urls that LS generates (/tasks/*/presign, /tasks/*/resolve, /projects/*/presign, /tasks/*/resolve). It changed the signature for presigned URLs and they threw CORS errors. 

# Screens

## Previous
![image](https://github.com/user-attachments/assets/280be08c-67e9-4e8c-ad5a-6eba9dbe0793)
![image](https://github.com/user-attachments/assets/53752052-860d-4268-a2da-b1959b22dfc2)

## Now

No more Baggage headers from sentry, no more GET + preflight in the network panel. 
![image](https://github.com/user-attachments/assets/91bd0a22-753c-4f65-8aee-71cc840be89e)
![image](https://github.com/user-attachments/assets/fbb0485e-a4a4-4544-8756-03c936ae2579)

# Issues
https://github.com/HumanSignal/label-studio/issues/6838
https://github.com/HumanSignal/label-studio/issues/7103
<!--

This description MUST be filled out for a PR to receive a review. Its primary purposes are:

 - to enable your reviewer to review your code easily, and
 - to convince your reviewer that your code works as intended.

Some pointers to think about when filling out your PR description:
 - Reason for change: Description of problem and solution
 - Screenshots: All visible changes should include screenshots.
 - Rollout strategy: How will this code be rolled out? Feature flags / env var / other
 - Testing: Description of how this is being verified
 - Risks: Are there any known risks associated with this change, eg to security or performance?
 - Reviewer notes: Any info to help reviewers approve the PR
 - General notes: Any info to help onlookers understand the code, or callouts to significant portions.

You may use AI tools such as Copilot Actions to assist with writing your PR description (see https://docs.github.com/en/copilot/using-github-copilot/using-github-copilot-for-pull-requests/creating-a-pull-request-summary-with-github-copilot); however, an AI summary isn't enough by itself. You'll need to provide your reviewer with strong evidence that your code works as intended, which requires actually running the code and showing that it works.

-->

makseq committed:
fix: BROS-4: Sentry adds Baggage header to presigned urls and it breaks CORS for storages. Fix Sentry.ts

makseq committed:
Fix

makseq committed:
Linter

bmartel commented on web/apps/labelstudio/src/config/Sentry.ts:
```diff
@@ -3,6 +3,7 @@ import * as ReactSentry from "@sentry/react";
 import type { RouterHistory } from "@sentry/react/build/types/reactrouter";
 import { Route } from "react-router-dom";
 import { isDefined } from "../utils/helpers";
+import { browserTracingIntegration } from "@sentry/react";
```
```suggestion
```

bmartel commented on web/apps/labelstudio/src/config/Sentry.ts:
```diff
@@ -14,10 +15,14 @@ export const initSentry = (history: RouterHistory) => {
+        /^\/api\/(?!projects\/\d+\/resolve)/,
+        /^\/api\/(?!projects\/\d+\/presign)/,
       ],
+      integrations: [browserTracingIntegration(), ReactSentry.reactRouterV5BrowserTracingIntegration({ history })],
```
```suggestion
      integrations: [ReactSentry.browserTracingIntegration(), ReactSentry.reactRouterV5BrowserTracingIntegration({ history })],
```

makseq committed:
Fix

makseq committed:
Fix

makseq committed:
FIx

makseq committed:
Fix

makseq committed:
Linter

makseq commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/15026215934)

makseq committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15026215934

makseq committed:
Merge branch 'develop' into 'fb-bros-4-2'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15026215934

makseq committed:
Revert pyproejct

makseq committed:
Merge branch 'fb-bros-4-2' of github.com:heartexlabs/label-studio into fb-bros-4-2

farioas committed:
Merge remote-tracking branch 'origin/develop' into fb-bros-4-2
